daprdocs/content/en/python-sdk-docs/python-client.md

```diff
@@ -196,8 +196,34 @@ async def executeConfiguration():
 asyncio.run(executeConfiguration())
 ```
 
-- For a full list of state operations visit [How-To: Get & save state]({{< ref howto-manage-configuration.md >}}).
 - Visit [Python SDK examples](https://github.com/dapr/python-sdk/tree/master/examples/configuration) for code samples and instructions to try out state management
 
+### Workflow
```

Comment:
This example is still very much in the early development stages and will improve/change over the next few days.

Reply:
Here's the updated version that is in master: https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py